src/hypofuzz/entrypoint.py

```diff
@@ -54,6 +55,12 @@
     nargs=-1,
     metavar="[-- PYTEST_ARGS]",
 )
+@click.option(
+    "--internal-backing-json-file",
+    type=click.Path(exists=True),
+    help="Back the dashboard display using a static json file. Written for the "
+    "example dashboard on hypofuzz.com. This option implies --dashboard-only.",
+)
```

Comment:
I don't think we should expose this as a user-facing option _unless_ we think it's also a useful feature for users, which... _maybe?_  If you want to share the results to someone else who has Hypofuzz, but without the database, or you want to archive the state of a run at some historical point?  I can see an argument, actually, but we should then document the user-facing reasons to use this.